src/olympia/lib/es/management/commands/reindex.py

```diff
@@ -171,66 +184,61 @@ def add_alias_action(action, index, alias):
         self.stdout.write('Building the task chain')
 
         to_remove = []
-        workflow = []
-
-        # For each alias, we create a new time-stamped index.
-        for alias, module in modules.items():
-            old_index = None
-
-            try:
-                olds = ES.indices.get_alias(alias)
-                for old_index in olds:
-                    # Mark the index to be removed later.
-                    to_remove.append(old_index)
-                    # Mark the alias to be removed from that index.
-                    add_alias_action('remove', old_index, alias)
-            except NotFoundError:
-                # If the alias dit not exist, ignore it, don't try to remove
-                # it.
-                pass
-
-            # Create a new index, using the alias name with a timestamp.
-            new_index = timestamp_index(alias)
-
-            # If old_index is None that could mean it's a full index.
-            # In that case we want to continue index in it.
-            if ES.indices.exists(alias):
-                old_index = alias
-
-            # Main chain for this alias: flag the database, then create the new
-            # index...
-            _chain = (
-                flag_database.si(new_index, old_index, alias) |
-                create_new_index.si(alias, new_index)
-            )
-            # ... Then start indexing data. gather_index_data_tasks() is a
-            # function returning a group of indexing tasks.
-            index_data_tasks = gather_index_data_tasks(alias, new_index)
-            if index_data_tasks.tasks:
-                _chain |= index_data_tasks
-
-            # Append that chain to the workflow we're going to execute.
-            workflow.append(_chain)
-
-            # Adding new index to the alias.
-            add_alias_action('add', new_index, alias)
-
-        # Group each alias chain so that they are executed in parallel if there
-        # is more than one alias to deal with.
-        workflow = group(workflow)
```

Comment:
Removal of the `for` loop and then this line is what fixes the issue, because then we no longer have nested groups/chords.